File: 168. Excel Sheet Column Title/C++/Solution.cpp
class Solution {
public:
    string convertToTitle(int n) {
        int i = 0;
        char c;
        string s;
        while (n) {
            n--;
            int t;
            t = n % 26;
            n = n / 26;
            c = 'A' + t;
            s = c + s;
        }
        return s;
    }
};

File: 168. Excel Sheet Column Title/Java/Solution.java
public class Solution {
    public String convertToTitle(int n) {
        StringBuilder builder = new StringBuilder();
        while (n != 0) {
            n--;
            builder.insert(0, (char)('A' + n % 26));
            n /= 26;
        }
        return builder.toString();
    }
}